torch/csrc/jit/passes/freeze_module.cpp

```diff
@@ -436,25 +454,30 @@ class AttributePropagator {
           }
           if (!paramConst) {
             auto attr = attrModule.attr(name);
+            auto type = attrModule.type();
 
             if (isEval) {
               attr = overrideGradient(attr);
             }
-            if (auto attrVal = tryInsertConstant(*graph, attr)) {
+
+            std::string fullName("self_");
+            for (auto& name : names_) {
+              fullName += name + '_';
+            }
+            fullName += name;
+            if (type->is_parameter(*type->findAttributeSlot(name))) {
```

Comment:
Can we also check for model buffers?
```
type->is_buffer(*type->findAttributeSlot(name)
```

Also, can we avoid inserting None attributes?